backend/src/cache/deployer/deploy-cache-service.sh

```diff
@@ -24,6 +24,17 @@ echo "Start deploying cache service to existing cluster:"
 
 NAMESPACE=${NAMESPACE_TO_WATCH:-kubeflow}
 mutatingWebhookConfigName="cache-webhook"
+
+# Should fail if there are connectivity problems
+# Gotcha: Listing all objects requires list permission, but when listing a single oblect kubecttl will fail if it's not found unless --ignore-not-found is specified.
+kubectl get mutatingwebhookconfigurations "${mutatingWebhookConfigName}" --namespace "${NAMESPACE}" --ignore-not-found >webhooks.txt
+
+if grep "${mutatingWebhookConfigName}" --word-regexp <webhooks.txt; then
+    echo "Webhook is already installed. Sleeping forever."
+    sleep 99999d
```

Comment:
Can we do `sleep infinity`?

Reply:
It does not work is every distribution, but in this case it works.
Done.